### PR TITLE
Fix nav issue after pressing "add more votes" on My votes step

### PR DESCRIPTION
**Pull request by LukeSugiura**
## Problem
When you navigate to step 2, press "Add more votes", then click android back button, you'll end up on Step 1 again. The android back button should navigate to step 2.

### CODE REVIEW

File: src/families/tron/VotingFlow/01-SelectValidator/index.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ type Props = {
     params: {
       accountId: string,
       transaction: Transaction,
+      fronStep2?: boolean,
```
Comment (LFBarreto) [inline src/families/tron/VotingFlow/01-SelectValidator/index.js:55]: typo here :pencil2:

File: src/families/tron/VotingFlow/01-SelectValidator/index.js [hunk 2]
```diff
@@ -123,6 +124,10 @@ function SelectValidator({ navigation }: Props) {
           })
         : transaction;
 
+    if (navigation.getParam("fromStep2")) {
+      navigation.pop(2);
```
Comment (LFBarreto) [inline src/families/tron/VotingFlow/01-SelectValidator/index.js:128]: I kind of understand the logic here but doesn't this make a nav glitch?

File: src/families/tron/VotingFlow/01-SelectValidator/index.js (hunk 1 above)
Comment (LukeSugiura) [reply src/families/tron/VotingFlow/01-SelectValidator/index.js]: Oops.
fix: https://github.com/LedgerHQ/ledger-live-mobile/pull/1221/commits/b0b1490138003073901ad5ef78539c27ae648a90

**Comment by gre** [comment]
@LukeSugiura please provide more info on how to test this.

**Comment by gre** [comment]
i prefer we hold this, there is no emergency to push this. QA to focus on rest

**Pull request merged**